apps/chat/README.md

```diff
@@ -9,7 +9,20 @@ The easiest way to get started with messaging in the Amazon Chime SDK is to depl
 - An [AWS account](https://aws.amazon.com/free/) that provides access to AWS services.
 - [Node.js](https://nodejs.org/en/download/) version 14 or higher.
 
-## Deploying the solution
+## Deploying the demo app
+
+You can run deploy.sh script in this folder to automatically deploy the demo app and it's backend.
+
+```bash
+cd apps/chat
+./deploy.sh
```

Comment:
Since the chat demo is an NPM project, I don't think it's good to add one more layer of deployment script. Instead, you could add a deploy script in the package.json.

- Customers know that this project is an NPM project.
- Writing the deploy script in Node.js will work in Node environments, while a bash script may not work on Windows. (We had several deployment issues on Windows.)

Reply:
Yes, my initial goal was to provide another simple way to deploy and test the demo app. Updated to use npm scripts instead.